### PR TITLE
[RUN-3243] Ignore requiresManualUpdate samples for runtime tests

### PR DESCRIPTION
**Pull request by Domiii**
* https://linear.app/replay/issue/RUN-3243/ignore-requiresmanualupdate-samples

### CODE REVIEW

File: packages/e2e-tests/scripts/get-stats.ts [hunk 1]
```diff
@@ -2,6 +2,17 @@ import { lstatSync, readFileSync, readdirSync } from "fs";
...
+
+/**
+ * Note: typeof does not include optional props, so we need to
+ * cast to this instead.
+ */
+type AllExampleProps = {
+  recording: string;
+  buildId: string;
+  playwrightScript?: string;
+  requiresManualUpdate?: string;
```
Comment (hbenl) [inline packages/e2e-tests/scripts/get-stats.ts:14]: Why do you treat this as `string`? It's a `boolean` in the `examples.json` file.

File: packages/e2e-tests/scripts/save-examples.ts [hunk 2]
```diff
@@ -116,6 +109,14 @@ async function saveRecording(
     `Saving ${chalk.grey.bold(example)} with recording id ${chalk.yellow.bold(recordingId)}`
   );
 
+  if (!skipUpload) {
+    await uploadRecording(recordingId, {
```
Comment (hbenl) [inline packages/e2e-tests/scripts/save-examples.ts:113]: Why did you move this down again? I had moved it up to make it work for node examples.

Comment (Domiii) [reply packages/e2e-tests/scripts/save-examples.ts:113]: Because of the `console.log` message. I'll try moving them both up - is that Ok?

File: packages/e2e-tests/scripts/get-stats.ts (hunk 1 above)
Comment (Domiii) [reply packages/e2e-tests/scripts/get-stats.ts:14]: Mistake

**Review by hbenl — APPROVED** [review]
